lib/version.c

```diff
@@ -153,6 +157,9 @@ char *curl_version(void)
 #endif
 #ifdef USE_GSASL
   char gsasl_buf[30];
+#endif
+#ifdef USE_OPENLDAP
+  char ldap_buf[30];
```

Comment:
Cant you declare the variable in the scope below?

Reply:
No, because it needs to remain in scope for the `src[]` array when it creates the final string just below this section.